### PR TITLE
Onsen UI 2.2.0 and Vue templates support for CLI

### PR DESCRIPTION
**Pull request by adamkozuch**
@frandiox @asial-matagawa  This blog post is about support CLI and LocalKit for Onsen UI and Vue 2. If you have any ideas for making it better or noticed some mistakes please let me know. Blog post should be published after release of templates. Right after release of templates I would like to add missing screenshots from LocalKit and CLI.

### CODE REVIEW

**Comment by asial-matagawa** [comment]
@adamkozuch 
Thank you!
I think the content of your article is great, but I would like to ask you to modify some words:
- `Angular2` -> `Angular 2+`
  - A space is required between `Angular` and its version number.
  - Google will release [Angular 4](https://github.com/angular/angular/blob/master/CHANGELOG.md) soon, so `Angular 2+` is preferred.
- `LocalKit` -> `Localkit`
  - The official name is `Localkit`.
- `Cssnext` -> `cssnext`
  - `cssnext` cannot be capitalized.

**Comment by adamkozuch** [comment]
@asial-matagawa Thank you for feedback. I corrected post.

**Comment by frandiox** [comment]
@adamkozuch Looks good! Maybe you want to remove this "the"? `but also other the transpirable projects`.
Also, perhaps you can add some info about Webpack. This is still using Webpack 1 but we will provide v2 support soon.

:+1:

**Comment by adamkozuch** [comment]
@frandiox thanks for feedback. I mentioned webpack 2 and corrected sentence with "the".

**Comment by adamkozuch** [comment]
I changed content of blog post. Now content is about release of CLI support for Onsen UI and Vue 2.

File: blog/posts/2017-03-21.markdown [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 author: adam
-date: 2017-03-27
+date: 2017-03-29
 id: vue-template-release
 title: "Monaca CLI support for Onsen UI 2.2.0 and Vue 2 binding"
```
Comment (frandiox) [inline blog/posts/2017-03-21.markdown]: @adamkozuch Can you remove "binding" from the title? Just `Monaca CLI support for Onsen UI 2.2.0 and Vue 2`.

Comment (adamkozuch) [inline blog/posts/2017-03-21.markdown]: @frandiox sure. Done.

**Pull request merged**